aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ListHandler.java

```diff
@@ -117,23 +135,27 @@ private DescribePatchBaselinesResponse describePatchBaselinesResponse(final Stri
                                                              final Logger logger) {
 
         final List<ResourceModel> models = new ArrayList<>();
+        List<Tag> tags = new ArrayList<Tag>();
+
         final List<String> baselineIdList = describePatchBaselinesResponse
                 .baselineIdentities()
                 .stream().map(baseline -> baseline.baselineId()).collect(Collectors.toList());
 
+        System.out.println(baselineIdList);
+
         for (String baselineId : baselineIdList) {
-            ResourceModel model = ResourceModel.builder()
-                    .id(baselineId)
-                    .build();
-            ResourceHandlerRequest<ResourceModel> requestFromId = ResourceHandlerRequest.<ResourceModel>builder()
-                    .desiredResourceState(model)
-                    .clientRequestToken(request.getClientRequestToken())
+
+            GetPatchBaselineRequest getPatchBaselineRequest = GetPatchBaselineRequest.builder()
+                    .baselineId(baselineId)
                     .build();
-            ProgressEvent<ResourceModel, CallbackContext> response
-                    = readHandler.handleRequest(proxy, requestFromId, null, logger);
 
-            ResourceModel resourceModel = response.getResourceModel();
-            models.add(resourceModel);
+            GetPatchBaselineResponse getPatchBaselineResponse =
+                    proxy.injectCredentialsAndInvokeV2(getPatchBaselineRequest, ssmClient::getPatchBaseline);
```

Comment:
My comment is not to use getPatchBaseline here directly,
My intention is to remove readHandler(getPatchBaseline call) completely from ListHandler, as customer are not expecting to get the full details of each baseline from ListHandler, but to get describe-patch-baselines return value, i.e. BaselineIdentities returned from describe-patch-baselines

Reply:
we will not return the full details.

Reply:
As you mentioned i have made the changes in Listhandler operation.
Now the ListHandler operation will return  the describePatchBaseline details of BaselineEntities and the response should be like as below

DescribePatchBaselinesResponse(BaselineIdentities=[
  PatchBaselineIdentity(BaselineId=arn: aws: ssm: us-east-1: 075727635805: patchbaseline/pb-03e3f588eec25344c,
  BaselineName=AWS-CentOSDefaultPatchBaseline,
  OperatingSystem=CENTOS,
  BaselineDescription=DefaultPatchBaselineforCentOSProvidedbyAWS.,
  DefaultBaseline=true)
]
]